darwin/dataset/release.py

```diff
@@ -22,6 +23,8 @@ class Release:
         The version of the ``Release``.
     name : str
         The name of the ``Release``.
+    status : str
```

Comment:
Unsure if it's common in darwin-py but this would  be better as an `enum` as it only has a select few values.